models/ed/DESCRIPTION

```diff
@@ -56,5 +59,6 @@ Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.1
```

Comment:
Please don't bump the Roxygen version in individual packages -- for CI sanity, we need to keep all packages built with the same version of Roxygen. If we need features from the newer Roxygen, we can bump it for all packages at once.

Reply:
It did seem like CI was running `testthat` 3rd edition, is that correct?  Is that cool to specify explicitly at the package level?

Reply:
Yes, testthat editions are fine to specify at the package level; those are separate from the Roxygen version.

Reply:
@Aariq you'll need to revert this line, and any other Rd file changes caused by running Roxygen 7.2.1, so that the CI results don't contain ["Version of roxygen2 last used with this package is 7.2.1.  You only have version 7.1.2"](https://github.com/PecanProject/pecan/runs/7528179653?check_suite_focus=true#step:8:127). This probably ought to trigger a CI failure, but meanwhile we're relying on catching it in manual review.

Reply:
Ah thanks, I had missed that one.